docs/Contract Creators/using-chainlink-price-reference-contracts.md

```diff
@@ -1,509 +0,0 @@
----
```

Comment:
redirect

Reply:
this exists somewhere else now?

Reply:
The price feeds are on the [LINK Token Contracts](https://docs.chain.link/docs/link-token-contracts/) page and an explanation of how to use them is in the [tutorials](https://docs.chain.link/docs/beginners-tutorial/) section.